src/client/sandbox/iframe.js

```diff
@@ -68,15 +68,14 @@ export default class IframeSandbox extends SandboxBase {
     }
 
     iframeReadyToInitHandler (e) {
-        // NOTE: Get and evaluate an iframe task script. The evaluation must be synchronous.
-        var msg = {
-            cmd:     COMMAND.getIframeTaskScript,
-            referer: settings.get().referer || this.window.location.toString()
-        };
-
-        syncServiceMsg(msg, iframeTaskScript => {
-            e.iframe.contentWindow.eval.apply(e.iframe.contentWindow, [iframeTaskScript]);
-        });
+        var iframeWithoutSrcTaskTemplate = settings.get().iframeWithoutSrcTaskTemplate;
+
+        iframeWithoutSrcTaskTemplate = iframeWithoutSrcTaskTemplate
+            .replace('{{{cookie}}}', this.cookieSandbox.getCookie())
+            .replace('{{{referer}}}', settings.get().referer || this.window.location.toString())
+            .replace('{{{iframeWithoutSrcTaskTemplate}}}', iframeWithoutSrcTaskTemplate.replace(/"/g, '\\"'));
+
+        e.iframe.contentWindow.eval.apply(e.iframe.contentWindow, [iframeWithoutSrcTaskTemplate]);
```

Comment:
Why we use the ```apply``` instead of the ```call``` function?
And i think that ```iframeWithoutSrcTaskTemplate``` is wrong variable name, may be use ```iframeWithoutSrcTask```?